web/core/components/issues/issue-layouts/kanban/block.tsx

```diff
@@ -217,7 +217,7 @@ export const KanbanIssueBlock: React.FC<IssueBlockProps> = observer((props) => {
             { "bg-custom-background-80 z-[100]": isCurrentBlockDragging }
           )}
           onClick={() => handleIssuePeekOverview(issue)}
-          disabled={!!issue?.tempId || isMobile}
+          disabled={!!issue?.tempId}
```

Comment:
Was it disabled earlier because it was was getting dragged on mobile? Please verify.